src/rp2040/Makefile

```diff
@@ -67,6 +68,7 @@ lib/rp2040_flash/rp2040_flash:
 	@echo "  Building rp2040_flash"
 	$(Q)make -C lib/rp2040_flash rp2040_flash
 
-flash: $(OUT)klipper.uf2 lib/rp2040_flash/rp2040_flash
+# Flash rules
```

Comment:
Dup'ing this comment looks odd.

-Kevin

Reply:
… not entirely sure how that snuck in, but it's gone now!